examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p><code>apply_transform(src='images', dst='images', func=lambda x: np.transpose(x, (1, 2, 0)).astype(int).astype(int)) # make it compatible with plt.imshow()</code></p><p> </p><p>what about making  plot_images_predictions compatible with pictures encoded as arrays by calling <code>plt.imshow(pic.T)</code> ? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
we won't get the correct image in the result:  `(c, x, y).T -> (y, x, c)`